tests/ffi/dev_mem_not_present/PURPOSE

```diff
@@ -0,0 +1,10 @@
+Verifies that /dev/mem is not present in QM partition.
```

Comment:
Please use this format in a next patch, I will unblock you for now.
 @pengshanyu  Also, your commit message container several duplicate signed-off-by (SOB) next time please remove the dup lines.

Title:
    Foobar 123 123

Description:
    Foobar 123 123

Input:
    Foobar 123 123

Expected Result:
    Foobar 123123

Jira:
    Foobar 123123

Reply:
Hi @dougsland, sorry, I didn't notice them. Sure, I'll pay attention next time. Thanks.

Reply:
Not your fault at all, we are starting to create it ;-)